external-apis/siopeplus.yaml

```diff
@@ -1452,5 +1459,266 @@ paths:
       summary: Download ack giornale di cassa
       tags:
       - Giornale di Cassa
+  /{idA2A}/PA/{codEnte}/disponibilita/:
+    get:
+      description: |
+        ENTE: Lista disponibilità liquide relative ad uno specifico Ente
+      parameters:
+      - $ref: '#/components/parameters/idA2A'
+      - $ref: '#/components/parameters/codEnte'
+      - $ref: '#/components/parameters/dataUploadDa'
+      - $ref: '#/components/parameters/dataUploadA'
+      - $ref: '#/components/parameters/download'
+      - $ref: '#/components/parameters/pagina'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FlussiPaginatiUpload'
+          description: Ricerca completata con successo
+        '400':
+          description: Bad Request
+        '401':
+          description: Identificativo A2A di chi origina la richiesta non ancora abilitato
+        '404':
+          description: Not Found
+        '406':
+          description: 'Assenza dell''header di richiesta `Accept: application/json`'
+        '429':
+          description: richieste più frequenti di quanto consentito dalle regole di
+            throttling
+          headers:
+            Retry-After:
+              $ref: https://teamdigitale.github.io/openapi/headers/v3.yaml#/Retry-After
+            X-RateLimit-Limit:
+              $ref: https://teamdigitale.github.io/openapi/headers/v3.yaml#/X-RateLimit-Limit
+            X-RateLimit-Remaining:
+              $ref: https://teamdigitale.github.io/openapi/headers/v3.yaml#/X-RateLimit-Remaining
+            X-RateLimit-Reset:
+              $ref: https://teamdigitale.github.io/openapi/headers/v3.yaml#/X-RateLimit-Reset
+        '503':
+          description: Servizio non disponibile
+          headers:
+            Retry-After:
+              $ref: https://teamdigitale.github.io/openapi/headers/v3.yaml#/Retry-After
+        default:
+          description: Errore non atteso
+      summary: Lista disponibilità liquide
+      tags:
+      - Disponibilità liquide
```

Comment:
Dovremmo correggere i tags in modo da avere solo ascii e senza spazi 
inserendo la descrizione nella definizione come nell'esempio qui sotto.

```
tags:
  - name: giornaleDiCassa
    description: Giornale di Cassa
  - name: flussoOrdinativi
    description: Flusso ordinativi
  - name: disponibilitaLiquide
    description: Disponibilità liquide
    externalDocs:
      url: link ad eventuale doc sul tema

```